Detectors/MUON/MCH/Workflow/src/TrackMCLabelFinderSpec.cxx

```diff
@@ -141,7 +143,7 @@ class TrackMCLabelFinderTask
         cluster.z = (cluster.z + trackRef.Z()) / 2.;
```

Comment:
L139 should be:
if (trackRef.getUserId() == deId) {

The rest looks fine to me.